app/screens/channel_info/channel_info.js

```diff
@@ -142,10 +143,12 @@ export default class ChannelInfo extends PureComponent {
                     theme={theme}
                 />
                 <Separator theme={theme}/>
+                {!isDirectMessage &&
                 <NotificationPreference
```

Comment:
Out of curiosity but would you be able to indent this? I've never seen it un-intended. Rather than having it aligned with `{!isDirectMessage` ?

Reply:
Not critical or anything. Just a preferential thing but if you deem no need, then we can ignore. Just one of those small things I like doing. Like this
![image](https://user-images.githubusercontent.com/17804942/94020765-4b0d6900-fd81-11ea-8977-fa53354a736a.png)